webapp/components/Editor/SuggestionList.vue

```diff
@@ -7,15 +7,15 @@
       :class="{ 'is-selected': navigatedItemIndex === index }"
       @click="selectItem(item)"
     >
-      {{ createItemLabel(item) }}
+      {{ createItemLabel(item) | truncate(50) }}
     </li>
     <template v-if="isHashtag">
-      <li v-if="!query" class="suggestion-list__item hint">
-        {{ $t('editor.hashtag.addLetter') }}
-      </li>
+      <li v-if="!query" class="suggestion-list__item hint">{{ $t('editor.hashtag.addLetter') }}</li>
       <template v-else-if="!filteredItems.find(el => el.id === query)">
         <li class="suggestion-list__item hint">{{ $t('editor.hashtag.addHashtag') }}</li>
-        <li class="suggestion-list__item" @click="selectItem({ id: query })">#{{ query }}</li>
+        <li class="suggestion-list__item" @click="selectItem({ id: query })">
+          #{{ query | truncate(50) }}
```

Comment:
:+1:

I just had to google what this does – very confusing if you don't know about Vue.js filters... Great opportunity to learn about them, though! So thanks for this! :smile:

Reply:
@mattwr18 and I learned this by @roschaefer ! 😍💫